src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/RegisterAktivteter.tsx

```diff
@@ -41,7 +41,7 @@ const RegisterAktiviteter: React.FC<{
         <VStack>
             <ExpansionCard
                 tittel={`Aktiviteter hentet fra Arena fra og med ${formaterNullableIsoDato(hentetInformasjon.fom)}`}
-                maxWidth={800}
+                maxWidth={1600}
```

Comment:
Er dette kanskje litt vel stort? Hvordan ville det sett ut? Burde noen celler deles?

Reply:
Lars har sett på dette og sagt det så fint ut, men jeg kan godt ta en runde til med han. Aksel-tabellen er ganske dynamisk "out-of-the-box", så dersom noen celler har veldig mye innhold så utvides de automatisk.

Reply:
![image (4)](https://github.com/user-attachments/assets/2f3064bf-6582-46bd-8d1e-cb8962f3b725)

Reply:
Setter max-bredden ned til 1200 pixler etter samtale med Lars: 9c04b027f92a828cd6e6e65a774de3e39c5e2c90